src/libsystemd/sd-event/sd-event.c

```diff
@@ -889,6 +890,10 @@ static void source_free(sd_event_source *s) {
         assert(s);
 
         source_disconnect(s);
+
+        if (s->type == SOURCE_IO && s->io.close)
+            safe_close(s->io.fd);
```

Comment:
nitpick: indent by multiples of 8 space please, see CODING_STYLE

Reply:
Yeah, I know that. I just let this one slip through the cracks. :)